python/nvtx/colors.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import functools
+
+_NVTX_COLORS = {
+    "green": 0x008000,
+    "blue": 0x0000FF,
+    "yellow": 0xFFFF00,
+    "purple": 0x800080,
+    "rapids": 0x7400FF,
```

Comment:
May want to remove this entry ;)
```suggestion
```

Reply:
This will make @jrhemstad sad :)

Reply:
:cry:

Reply:
Can we somehow extend this list of colors in cuDF to add this?

Reply:
> Can we somehow extend this list of colors in cuDF to add this?

Yes we can.